### DISCUSSION TITLE
Please force-enable the Google Search dark mode option

### DISCUSSION THREAD

**Discussion by Void48** [Floorp-Projects/Floorp#564] [Feature Request]
I'm always frustrated when I google something in Incognito mode and the google search blinds me because it always defaults to light mode instead of dark mode.

The default option that Google Search uses when you search for something in Incognito mode is: **Device default**
But I would like it to use **Dark theme: on** always

**Comment by bytenekos** [comment]
Would be better if it's an option as I use light mode and dark and switch between them

**Comment by Ryosuke-Asano** [comment]
Submit your issue to Google
